torch/csrc/jit/runtime/serialized_shape_function_registry.cpp

```diff
@@ -421,8 +478,7 @@ def mm(self: List[int],
     ops.prim.RaiseException("AssertionError: ")
   return [self[0], mat2[1]]
 
-)=====")
-+ std::string(R"=====(def dot(self: List[int],
+def dot(self: List[int],
```

Comment:
These changes should no longer be necessary due to the this commit I landed:
https://github.com/pytorch/pytorch/pull/83092

If you need to manually edit the generated file for any reason, let me know, and I can look into a fix for it.

Reply:
Hi I rebased on top of my branch using the following command
```
git pull --rebase upstream viable/strict
```
re-ran gen_jit_shape_functions.py but I don't see any changes to the serialized_shape_function_registry.cpp file.

Reply:
Did you rebuild Pytorch first?

Reply:
The changes to `sum_mean_dim` for example should no longer be happening. What happens if you delete `serialized_shape_function_registry.cpp` and rerun gen_jit_shape_functions.py?

Reply:
yes

Reply:
hi I have re-ran and pushed the changes but the changes to sum_mean_dim still seem to be happening.